Tests/TokamakStaticHTMLTests/RenderingTests/ShapeTests.swift

```diff
@@ -65,6 +65,123 @@ final class ShapeRenderingTests: XCTestCase {
       timeout: defaultSnapshotTimeout
     )
   }
+
+  func testTangentArc() {
```

Comment:
When running this test, the call stack fills in `getArc` where it gets the second part of the chunk: https://github.com/TokamakUI/Tokamak/blob/512933bcbf7232a023a9d40e72801bb1a830602a/Sources/TokamakCore/Shapes/Path/PathMutations.swift#L189
It seems to make sense, as the `chunk1` angle is created by adding `.pi / 2` to the startAngle, which the call to `getArc` then checks and sees the value is greater than `.pi / 2`.

Personally I'm not sure what the solution to this is, I'm hoping you have some ideas 🙃